### ISSUE TITLE
axios.CancelToken is waiting for delay specified in delayResponse

### ISSUE THREAD

**Issue by chestozo** [ctimmerm/axios-mock-adapter#258]
Hi there!
First of all - thank you for a great tool! )

I am using `MockAdapter` in storybook environment during development phase.
I have a form where I am performing server form field validation.
For that to look more real I am using a combination of:
```
const apiMock = new MockAdapter(axiosInstance, { delayResponse: 5000 })
const token = axios.CancelToken.source()
//...
token.cancel('Operation cancelled')
```

In real life `token.cancel` will lead to almost instant reaction of a request been aborted.
While in my case with a `MockAdapter` I need to wait for the `5000`ms to complete and only after that I get a request failure.

What do you think of an option to abort request immediately based on `axios.CancelToken`?

**Comment by ctimmerm** [comment]
Yeah, that makes sense.  `delayResponse` should not affect canceling a request.

**Comment by imsunhao** [comment]
hello
Is there a plan to solve this problem now? I've also encountered this problem.
